src/devices/LidarLiteV3/MeasurementRepetitionMode.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.TimeOfFlight.Models.LidarLiteV3
+{
+    /// <summary>
+    /// Measurement repetition modes
+    /// </summary>
+    public enum MeasurementRepetitionMode
+    {
+        /// <summary>
+        /// Disabled, measurements are done once per command.
+        /// </summary>
+        Off,
+
+        /// <summary>
+        /// Measurements are done repetitively per command defined in OUTER_LOOP_COUNT.
+        /// </summary>
+        Repeat,
+
+        /// <summary>
+        /// Measurements are done infinitely.
```

Comment:
Can you explain also on example or when you should use it? Is this equivalent to: Start measuring and finish measuring when on next read?

Reply:
Actually briefly convered it here, https://github.com/vornet/iot/tree/LidarLiteV3/src/devices/LidarLiteV3/samples#repetition-mode.  It can internally trigger measurements, either in a finite loop or infinitely.  So it's the equivalent of doing the measurement in a c# loop, but on the device itself with more predicable timing since there's less overhead.